libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -103,7 +104,7 @@ template <class T>
 void HarmonicNotchFilter<T>::create(uint8_t harmonics)
 {
     _num_filters = 1;
-    for (uint8_t i = 0; i < HNF_MAX_FILTERS-1; i++) {
+    for (uint8_t i = 0; i < HNF_MAX_HARMONICS && _num_filters < HNF_MAX_FILTERS; i++) {
```

Comment:
Actually since we are now limiting the number of filters I can just change the docs to allow the 8th bit. Also pretty sure I get 3 filters above - yes I only get 2 extra harmonics but with the fundamental it's 3 total.

Reply:
> Actually since we are now limiting the number of filters I can just change the docs to allow the 8th bit.

Just changing the docs doesn't work though because other places only allow 8 (including the first harmonic).

> Also pretty sure I get 3 filters above - yes I only get 2 extra harmonics but with the fundamental it's 3 total.

You are right!

Reply:
Ok as per @lthall all harmonics can now be controlled. I have made the default the first two.